contributors/devel/release.md

```diff
@@ -0,0 +1,315 @@
+# Targeting Features, Issues and PRs to Release Milestones
+
+This document is focused on Kubernetes developers and contributors
+who need to create a feature, issue, or pull request which targets a specific
+release milestone.
+
+-   [TL;DR](#tl-dr)
+-   [Definitions](#definitions)
+-   [The Release Cycle](#the-release-cycle)
+-   [Removal Of Items From The Milestone](#removal-of-items-from-the-milestone)
+-   [Adding An Item To The Milestone](#adding-an-item-to-the-milestone)
+    -   [Milestone Maintainers](#milestone-maintainers)
+    -   [Feature additions](#feature-additions)
+    -   [Issue additions](#issue-additions)
+    -   [PR Additions](#pr-additions)
+-   [Other Required Labels](#other-required-labels)
+    -   [SIG Owner Label](#sig-owner-label)
+    -   [Priority Label](#priority-label)
+    -   [Issue Kind Label](#issue-kind-label)
+
+The process for shepherding features, issues, and pull requests
+into a Kubernetes release spans multiple stakeholders:
+* the feature, issue, or pull request owner
+* SIG leadership
+* the release team
+
+Information on workflows and interactions are described below.
+
+As the owner of a feature, issue, or pull request (PR), it is your
+responsibility to ensure release milestone requirements are met.
+Automation and the release team will be in contact with you if
+updates are required, but inaction can result in your work being
+removed from the milestone.  Additional requirements exist when the
+target milestone is a prior release (see [cherry pick
+process](cherry-picks.md) for more information).
+
+## TL;DR
```

Comment:
This is really close, but I'm still a little confused:
- what is this table describing? ie: "tl;dr If you want your PR to get merged, it needs the following required labels and milestones, represented here by the `/commands` it would take to add them"
- is this for PR's only, or also for issues?
- ... because issues don't need /lgtm and /approved

Reply:
Yes.  Added hopefully clarifying text.